src/main/java/frc/robot/maps/subsystems/ArmMap.java

```diff
@@ -12,6 +12,8 @@ public class ArmMap {
     public SmartMotorController extendMotor;
     public double softMaxDistance;
     public double softMinDistance;
+    public double hardMaxDistance;
```

Comment:
We should add the arm pivot height to the map instead of specifying it in the subsystem.